DXRMapFixups/DeusEx/Classes/DXRFixupM02.uc

```diff
@@ -359,9 +359,16 @@ function PostFirstEntryMapFixes()
 
 function AnyEntryMapFixes()
 {
+    local ConEventSpeech ces;
     local Jock j;
 
     switch (dxr.localURL) {
+    case "02_NYC_STREET":
+        ces = GetSpeechEvent(GetConversation('SmugglerDoorBellConvo').eventList, "...");
```

Comment:
I think the search text should be more specific, to make sure it doesn't overwrite a different language or anything

Reply:
That's 9 extra equality checks! Anyway, done.